target/linux/ath79/nand/base-files/lib/upgrade/platform.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: GPL-2.0
+#
+# Copyright (C) 2019 Jeff Kletsky
+#
```

Comment:
Same ...

Reply:
(previously discussed)

There is absolutely no reason why a NOR-based device should have to chew up flash space for this file. You highlighted one of the reasons to use `ath79/nand/base-files/` over `ath79/base-files/`, or `package/base-files/files/` for that matter.